ui/app/routes/vault/cluster/access/oidc/providers/provider/clients.js

```diff
@@ -3,6 +3,6 @@ import Route from '@ember/routing/route';
 export default class OidcProviderClientsRoute extends Route {
   async model() {
     const { allowedClientIds } = this.modelFor('vault.cluster.access.oidc.providers.provider');
-    return await this.store.query('oidc/client', { filterIds: allowedClientIds });
```

Comment:
before, we were just filtering a list of strings (the `keys` of the `LIST` response), but because `clientIds` aren't the same as the client's `id`, we need to pass the attr name we want to filter for, along with the values we want to match